### ISSUE TITLE
[bug] kube-proxy not upgraded to v1.26.4

### ISSUE THREAD

**Issue by gerhard** [siderolabs/omni-feedback#25]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

On Omni `v0.6.2`, I triggered a K8s upgrade from `v1.26.3` to `v1.26.4`. All components except `kube-proxy` got upgraded correctly:

```console
> omnictl get rcmc -o yaml | rg v1.26
                image: ghcr.io/siderolabs/kubelet:v1.26.4
                image: registry.k8s.io/kube-apiserver:v1.26.4
                image: registry.k8s.io/kube-controller-manager:v1.26.4
                image: registry.k8s.io/kube-proxy:v1.26.2
                image: registry.k8s.io/kube-scheduler:v1.26.4
```

### Expected Behavior

`kube-proxy` should be upgraded to `v1.26.4` too.

### Steps To Reproduce

Given a cluster running K8s `v1.26.4`, click the upgrade K8s to latest button in Omni.

**Comment by smira** [comment]
The fix will come in the next version of Omni, at the moment it doesn't update the bootstrap manifests (yet), thanks for reporting!

**Comment by gerhard** [comment]
OK, so after Omni `v0.7.0`. No worries. Keeping this one open then. We are down to just `1` issue then. Great job!

FWIW, I am very close to rolling this in a production cluster. I am aiming for this week. Looking forward to it!

**Comment by gerhard** [comment]
I can confirm that this is now fixed in Talos `v1.4.0` managed by Omni `v0.7.1`:

```console
omnictl get rcmc -o yaml | rg image

                image: ghcr.io/siderolabs/kubelet:v1.26.4
                image: ghcr.io/siderolabs/installer:v1.4.0
                image: registry.k8s.io/kube-apiserver:v1.26.4
                image: registry.k8s.io/kube-controller-manager:v1.26.4
                image: registry.k8s.io/kube-proxy:v1.26.4
                image: registry.k8s.io/kube-scheduler:v1.26.4
```

Thank you @smira 💪
